indico/modules/events/surveys/templates/survey_announcement.html

```diff
@@ -0,0 +1,28 @@
+{% set survey_count = surveys|length %}
```

Comment:
`len()` on a list is `O(1)`, so i wouldn't cache it. but that's just my personal opinion so i'm fine either way